index/templates/index/index.html

```diff
@@ -28,22 +29,34 @@ <h2 class="fs-1 fw-bolder">Welcome to United Events</h2>
         </div>
     </section>
 
-
     <section class="container my-5">
         <div class="row">
-            <div class="col">
-                <p class="fs-5">
-                    Connecting, Supporting, Celebrating, and Participating
-                    Our platform is an empowering hub for the LGBTQIA+ community, providing a personalizable space to derive the best individual experience and benefit from major Pride events throughout the UK in Summer 2024. We offer information on external resources, venues, and approved safe spaces, all promoting diversity and inclusion.
-                    Features
-                    Empowering Hub: Personalize your experience to get the most out of Pride events.
-                    Resource Links: Access support groups, educational information and safe spaces.
-                    Dynamic and Supportive Community: Celebrate diversity, feel seen, heard, and valued.
-                    Real-World Impact: Our platform aims to make a positive change in individual lives.
-                    Join us to experience the vibrant, supportive, and dynamic environment that celebrates and empowers the LGBTQIA+ community.
-                </p>
-            </div>
+          <div class="col">
+            <p class="fs-5">
+              Our platform is an empowering place for the LGBTQIA+ community, providing a personalizable space to derive the best individual experience and benefit from major Pride events throughout the UK in Summer 2024. We offer information on external resources, venues, and approved safe spaces, all promoting diversity and inclusion.
+            </p>
+            <h5><br>Features:</h5>
+            <ul class="list">  <li>
+                <span class="bullet"></span>
```

Comment:
What are all of these `<span class="bullet"></span>` elements? I can't see any request for a CSS class of `bullet` anywhere, and it's not a Bootstrap class, so it's not doing anything?

Same for the parent `<ul class="list">`, what is the `list` class? I can't see it in your CSS code.

Reply:
It was me trying to find the best & right way to achieve the aesthetic which you seem to have understood based on what you say below PR52 delivered. Maybe there were remnant orphans from me trying SO hard to figure it all out?!